### PR TITLE
Add recipe for nc2zarr

### PR DESCRIPTION
**Pull request by pont-us**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bcdev** [comment]
I am willing to be listed as a maintainer.

**Comment by forman** [comment]
I am willing to be listed as a maintainer.

**Comment by AliceBalfanz** [comment]
I am willing to be listed as a maintainer.

**Comment by TonioF** [comment]
I am willing to be listed as a maintainer.

**Comment by dzelge** [comment]
I am willing to be listed as a maintainer.

**Comment by pont-us** [comment]
I am willing to be listed as a maintainer.

**Comment by pont-us** [comment]
@conda-forge/help-python ready for review :).

File: recipes/nc2zarr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest tests
+    - nc2zarr --help
+
+about:
+  home: https://github.com/simplejson/simplejson
```
Comment (kmuehlbauer) [inline recipes/nc2zarr/meta.yaml:49]: Please add the correct link here.

File: recipes/nc2zarr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A Python tool that converts NetCDF files to Zarr format'
+  description: |
+    nc2zarr reads one or more NetCDF datasets and writes or appends them
+    to a single Zarr output dataset.
+  doc_url: https://github.com/simplejson/simplejson/README.md
```
Comment (kmuehlbauer) [inline recipes/nc2zarr/meta.yaml:57]: Here another two lines to fix

**Review by kmuehlbauer — CHANGES_REQUESTED** [review]

**Comment by kmuehlbauer** [comment]
@pont-us Please add the changes to the metadata in the feedstock, when it is created.

**Comment by pont-us** [comment]
> @pont-us Please add the changes to the metadata in the feedstock, when it is created.

Thank you, will do!

**Pull request merged**